### PR TITLE
Conan 1.19 write-up

### PR DESCRIPTION
**Pull request by danimtb**

### CODE REVIEW

File: _posts/2019-10-01-New-conan-release-1-19.markdown [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+You can read about how to manage Apple frameworks in the [documentation](https://docs.conan.io/en/latest/howtos/link_apple_framework.html).
+
+## Enabling and disabling remotes
+
+Conan uses a decentralized model in order to retrieve and share your packages. This means that you can have different remotes configured in
+the client and choose which one to install packages from or upload those to. However, sometimes remotes are not available for connection
+(working behind a proxy for example) when a `conan install` is done and it fails.
+
+With `conan remote disable <remote>` you can now disable remotes the remotes that you don't want to use for an installation. The remote will
```
Comment (SSE4) [inline _posts/2019-10-01-New-conan-release-1-19.markdown]:
```suggestion
With `conan remote disable <remote>` you can now disable  the remotes that you don't want to use for an installation. The remote will
```

**Comment by uilianries** [comment]
Preview:

Screenshot_2019-10-01 Conan 1 19 Better integration for CMake and Pkg-Config, OSX frameworks support, enabling and disablin

File: _posts/2019-10-01-New-conan-release-1-19.markdown [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+## Support for Apple OSX frameworks
+
+The second feature is related to OSX frameworks, where libraries can generate files such as `sfml-system.framework` that have to be consumed
+with the `-framework` flag. Some recipes were declaring flags to link with frameworks libraries using `cpp_info.exelinkflags` and
+`cpp_info.sharedlinkflags` but this is not the right place to declare this information.
+
+So this Conan release is including two more variables to `cpp_info` dedicated for this purpose:
+
+- `cpp_info.frameworks`: List containing the names of the frameworks created by the package.
```
Comment (uilianries) [inline _posts/2019-10-01-New-conan-release-1-19.markdown]:
```suggestion
- `cpp_info.frameworks`:A list containing the names of the frameworks created by the package.
```

File: _posts/2019-10-01-New-conan-release-1-19.markdown [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+- `cpp_info.framework_paths`: List of the relative directories to package folder used to locate the framework binaries (Default to
```
Comment (uilianries) [inline _posts/2019-10-01-New-conan-release-1-19.markdown]:
```suggestion
- `cpp_info.framework_paths`: A list of the relative directories to package folder used to locate the framework binaries (Default to
```

**Comment by czoido** [comment]
Lots of cool things for 1.19! :smiley:

**Pull request merged**